source/mainnet/net/resources/glossary.rst

```diff
@@ -493,6 +500,13 @@ Qualified authority
 
 A governmental body that has the authority to act in a relevant jurisdiction. For example, a local police force, a local court or an investigatory division of a local authority that regulates financial conduct may have authority to act in their relevant jurisdictions. These authorities are qualified to begin the process of revoking the anonymity of a user when they proceed through established legal channels and make a formal request. The outcome of such a request is likely to be that a qualified authority obtains an official order, which may be in the form of a warrant, court order, or similar instrument. Only after a qualified authority validly serves an official order upon the relevant :ref:`anonymity revokers<glossary-anonymity-revoker>` and :ref:`identity provider<glossary-identity-provider>`, can the real-world identity of a user be revealed and only to the extent set out in the order.
 
+.. _glossary-quorum-certificate:
+
+Quorum certificate
+==================
+
+When the members of the finalization committee sign the block, their collective signatures are aggregated to form a quorum certificate. This quorum certificate is then included in the next block. If the leader fails to produce a block in the round, or not enough signatures were gathered for a quorum certificate, then the finalizers will instead send timeout messages, which are aggregated to form a timeout certificate. Each block either contains a quorum certificate or a timeout certificate for the previous round.
```

Comment:
A block always contains a quorum certificate as it serves as a reference to the parent block. The block _might_ contain a timeout certificate if the previous round timed out.

Reply:
A quorum certificate (or a timeout certificate) is what makes sure that the protocol progresses. When a node sees a valid quorum certificate it progresses to the next round (and likewise for the timeout certificate).